libs/ui-toolkit/src/components/key-value-table/key-value-table.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import * as React from 'react';
+
+export interface KeyValueTableProps
+  extends React.HTMLAttributes<HTMLTableElement> {
+  title?: string;
+  children: React.ReactNode;
+  muted?: boolean;
+  numerical?: boolean;
+}
+
+export const KeyValueTable = ({
+  title,
+  children,
+  className,
+  muted,
+  numerical,
+  ...rest
+}: KeyValueTableProps) => {
+  return (
+    <React.Fragment>
+      {title && <h3 className="mt-2.5 mb-1.25">{title}</h3>}
+      <table
+        data-testid="key-value-table"
+        {...rest}
+        className={`w-full border-collapse mb-2.5 [border-spacing:0] break-all ${
```

Comment:
Chore (blocking): Our custom Tailwind theme means the values you've used here with decimals won't work. The currently allowed spacings are defined in libs/tailwindcss-config/src/theme.js. If you need a custom value, use the square bracket syntax, i.e. mb-[0.3rem]. 

Question: not seen the syntax for what you've done here with `[border-spacing:0]`. What does this do?

Reply:
Ah that's good to know, thanks!